be/src/exec/parquet/encoding_plain.h

```diff
@@ -53,6 +56,30 @@ class PlainEncoder<Slice> final : public Encoder {
     faststring _buffer;
 };
 
+template <>
+class PlainEncoder<bool> final : public Encoder {
```

Comment:
Does the problem is boolean type in parquet, StarRocks do not support it?
I can not get the point from the title and commit message.